### PR TITLE
Work In Progress, first version of lodesiempre.com

### PR DESCRIPTION
**Pull request by juanazadian**
first version of the website,
includes changes in lodesiempre.html and style.css

### CODE REVIEW

File: css/lodesiempre.html [hunk 1]
```diff
@@ -19,24 +17,168 @@
 
   <div class="header-background">
```
Comment (jpbalarini) [inline css/lodesiempre.html]: here you can use the `<header>` tag

File: css/lodesiempre.html [hunk 2]
```diff
@@ -1,14 +1,12 @@
 <!DOCTYPE html>
-<html>
+<html >
 
 <head>
```
Comment (jpbalarini) [inline css/lodesiempre.html:5]: let's add the `<meta charset="utf-8">` to the header, so browsers can detect the encoding (that was what was happening on Safari)

File: css/lodesiempre.html [hunk 3]
```diff
@@ -19,24 +17,168 @@
...
+        <h4 class="secondary-font white-color margin-subtitle">Compra hoy un voucher con descuento, y ayudá a tu comercio favorito a surfear la ola.</h4>
         <h1 class="primary-font primary-color margin-title">417 personas</h1>
-        <h3 class="secondary-font white-color margin-subtitle"> ya apoyaron esta causa</h3>
-        <h2 class="primary-font primary-color margin-subtitle">80 restaurantes adheridos</h2>
+        <h4 class="secondary-font white-color margin-subtitle"> ya apoyaron esta causa</h4>
+        <h3 class="primary-font primary-color margin-subtitle">80 restaurantes adheridos</h3>
+    </div>
+  </div>
+
+  <div class="row-container">
```
Comment (jpbalarini) [inline css/lodesiempre.html]: we can have a `<main>` tag that contains all the main content, and inside that multiple `<section>`s

File: css/lodesiempre.html [hunk 4]
```diff
@@ -19,24 +17,168 @@
...
+    <img src="onboarding-assets/img/donacion.png" class="third-width column-container">
+    <div class="column-container left-align-flex third-width">
+      <h2 class="primary-font primary-color margin-subtitle">Por cada voucher vendido</h2>
+      <h1 class="primary-font secondary-color margin-subtitle">FNC donara 1 vianda</h1>
+      <h3 class="primary-font primary-color margin-subtitle">a instituciones seleccionadas.</h3>
+      <h5 class="secondary-font body-color margin-button"> Nos importa la salud y el impacto socio economico en la comunidad. El proposito de este proyecto es darnos aliento, que codo con codo salimos adelante.</h5>
```
Comment (jpbalarini) [inline css/lodesiempre.html]: this is more a paragraph `<p>` than a title element. Check below all that are similar

File: css/styles.css [hunk 5]
```diff
@@ -80,10 +143,118 @@ h1 {
...
+  padding: 35px 59px 35px 60px;
+  border-radius: 10px;
+  box-shadow: 0 3px 6px 0 rgba(0, 0, 0, 0.16);
+  border: solid 1px #665eff;
+  background-color: #ffffff;
+  text-align:center
+}
+
+.grey-box {
+
```
Comment (jpbalarini) [inline css/styles.css]: check this empy line

File: css/lodesiempre.html [hunk 6]
```diff
@@ -19,24 +17,168 @@
...
+  <div class="banner-medium-background column-container">
+    <h2 class="secondary-font secondary-color margin-subtitle"><b>NOS APOYAN EN LA CAUSA</b></h2>
+    <div class="row-container">
+      <img class="brand-container margin-box" src="onboarding-assets/img/apoyan/eagerworks.png">
+      <img class="brand-container margin-box" src="onboarding-assets/img/apoyan/fnc.png">
+      <img class="brand-container margin-box" src="onboarding-assets/img/apoyan/yendo.png">
+    </div>
+  </div>
+
+  <div class="footer-background">
```
Comment (jpbalarini) [inline css/lodesiempre.html]: this could be a `<footer>` element

**Review by jpbalarini — CHANGES_REQUESTED** [review]

File: css/lodesiempre.html (hunk 4 above)
Comment (juanazadian) [reply css/lodesiempre.html]: if I define all elements as a `<p>` how do I set the font-size accordingly without using a bunch of extra classes?
Isn't it better if I use h1..5 with the font-sizes that I previously defined?

File: css/lodesiempre.html [hunk 7]
```diff
@@ -1,42 +1,193 @@
...
-
-  <div class="row-container">
-    <img src="onboarding-assets/img/donacion.png" style="width: 33%;">
-    <div class="column-container" style="width: 33%;">
-      <h2 class="primary-font primary-color margin-title">Por cada voucher vendido</h3>
-      <h1 class="primary-font secondary-color margin-subtitle">FNC donará 1 vianda</h1>
-      <h2 class="primary-font primary-color margin-subtitle">a instituciones seleccionadas.</h3>
-      <h5 class="secondary-font body-color margin-title"> Nos importa la salud y el impacto socio económico en la comunidad. El propósito de este proyecto es darnos aliento, que codo con codo salimos adelante.</h5>
+    <div class="header-background column-container">
+        <h1 class="primary-font white-color margin-subtitle">Apoyá a los comercios gastronómicos.</h1>
```
Comment (jpbalarini) [inline css/lodesiempre.html]: check indentations, seems to be using 4 spaces instead of 2

File: css/lodesiempre.html [hunk 8]
```diff
@@ -1,42 +1,193 @@
...
+
+    <section class="banner-large-background">
+      <section class="column-container ">
+        <h3 class="secondary-font secondary-color margin-subtitle"><b>¿COMO FUNCIONA?</b></h3>
+        <section class="row-container margin-button text-align-center">
+          <section class="column-container width-cap grey-box small-padding">
+              <section class="round-button third-background-color">
+                  <h4 class="secondary-font white-color margin-subtitle"> <b>1</b></h4>
+              </section>
+              <p class="font-size-medium secondary-font body-color margin-subtitle"> <b>Elegi la forma que <br> deseas colaborar.</b> </p>
```
Comment (jpbalarini) [inline css/lodesiempre.html:76]: Remove empty space `margin-subtitle"><b>`

**Review by jpbalarini — APPROVED** [review]

**Pull request merged**